### PR TITLE
84 Add UI components

### PR DESCRIPTION
**Pull request by ntermpmm**
- [x] add homepage components
- [] add styling

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,45 +1,49 @@
...
+                    {" | "}
+                    <a
+                        className="App-link"
+                        href="https://vitejs.dev/guide/features.html"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        Vite Docs
+                    </a>
+                    {console.log("From 84")}
```
Comment (chanyaya) [inline src/App.jsx:42]: REMOVE THIS CONSOLELOG EDOK

Comment (ntermpmm) [reply src/App.jsx:42]: WHYYYY CONSOLE LOG IS not bad

Comment (chanyaya) [inline src/App.jsx:42]:
```suggestion
                   
```

**Review by ToEzBit — APPROVED** [review]
thum dee

**Pull request merged**